cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/DocumentManager.cs

```diff
@@ -292,17 +292,16 @@ public Document UpdateClaimDmer(Guid loginId, Guid documentId)
                 Success = false
             };
 
+            var loggedinUser = dynamicsContext.dfp_logins.Where(login => login.dfp_loginid ==  loginId).FirstOrDefault();
+
             var querydocument = dynamicsContext.bcgov_documenturls
             .Expand(doc => doc.dfp_DocumentTypeID)
-            .Where(doc => doc.dfp_DocumentTypeID.dfp_code == _configuration["CONSTANTS_DOCUMENT_TYPE_DMER"] && doc.bcgov_documenturlid == documentId).FirstOrDefault();
-
-           
-                if (querydocument != null)
-                {
-                    querydocument._dfp_loginid_value = loginId;
-                }
-
-            dynamicsContext.UpdateObject(querydocument);
+            .Expand(doc => doc.dfp_LoginId)
+            .Where(doc => doc.dfp_DocumentTypeID.dfp_code == _configuration["CONSTANTS_DOCUMENT_TYPE_DMER"] && doc.bcgov_documenturlid == documentId ).FirstOrDefault();
+            if (querydocument != null && loggedinUser != null)
+            {
+              dynamicsContext.SetLink(querydocument, nameof(bcgov_documenturl.dfp_LoginId), loggedinUser);     
+            }
             dynamicsContext.SaveChanges();
             dynamicsContext.DetachAll();
```

Comment:
I think you want to remove this DetachAll

Reply:
Fixed and restested the Claim Query